controllers/ControllerComment.php

```diff
@@ -94,8 +90,8 @@ public function checkComment(string $comment, int $commentId, int $postId){
 	public function deleteComment(){
 
 		$key = (new Session)->getKey('post');
-		$this->_commentManager->deleteComment($this->_method[3]);
-		header('location: http://'.$this->_server.'/Post/'. $key['id']);
+		$this->_commentManager->deleteComment($this->_slug);
+		header('location: http://'.$this->_server.'/Post/viewPost/'. $key['id']);
```

Comment:
Codacy found an issue: [The use of function header() is discouraged](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6061935)